runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/AbstractWindmillStream.java

```diff
@@ -252,7 +249,7 @@ public final Instant startTime() {
     return new Instant(startTimeMs.get());
   }
 
-  private class ResponseObserver implements StreamObserver<ResponseT> {
+  public class ResponseObserver implements StreamObserver<ResponseT> {
```

Comment:
ditto, seems like this could remain private?

Reply:
done.